src/components/ForgotPassword.html

```diff
@@ -0,0 +1,75 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<link
+			rel="stylesheet"
+			href="../styles/root.css"
+		/>
+	</head>
+	<body>
+		<div
+			class="inputDomMainContainer flex-column-center"
+		>
```

Comment:
indentation can be fixed. Currently it is taking lot more space

Reply:
Okay, I'll Do that